### PR TITLE
docs: hexo-generator-tag v1, hexo-renderer-ejs v1 & hexo-renderer-stylus v1

### PR DESCRIPTION
**Pull request by curbengh**
## Check List

**Please read and check followings before submitting a PR.**

- [x] Others (Update, fix, translation, etc...)

### CODE REVIEW

File: source/_posts/2019-09-06-hexo-generator-tag-1_hexo-renderer-ejs-1_hexo-renderer-stylus-1.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+### Breaking change
+-  chore: require at least node 8 [#25]
+
+### Feature
+- Add the "order_by" option [#22]
+
+### Refactor
+- refactor(package): use Object.assign instead of object-assign package [#14]
+- refactor(ES2015): use let/const & arrow-return [#16)
```
Comment (YoshinoriN) [inline source/_posts/2019-09-06-hexo-generator-tag-1_hexo-renderer-ejs-1_hexo-renderer-stylus-1.md]: markdown link syntax is wrong

```diff
- refactor(ES2015): use let/const & arrow-return [#16)
+ refactor(ES2015): use let/const & arrow-return [#16]
```

File: source/_posts/2019-09-06-hexo-generator-tag-1_hexo-renderer-ejs-1_hexo-renderer-stylus-1.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+## hexo-renderer-stylus 1.0.0
+
+### Breaking change
+- chore: require at least node 8 [#33]
+
+### Fix
+- upgrade dependencies and make it be compatible with node v6 [#13]
+
+### Refactor
+- refactor: es6-fy [#34)
```
Comment (YoshinoriN) [inline source/_posts/2019-09-06-hexo-generator-tag-1_hexo-renderer-ejs-1_hexo-renderer-stylus-1.md]:
```diff
- refactor: es6-fy [#34)
+ refactor: es6-fy [#34]
```

File: source/_posts/2019-09-06-hexo-generator-tag-1_hexo-renderer-ejs-1_hexo-renderer-stylus-1.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+- refactor: use Object.assign and var to let/const [#8]
+- chore(eslint): use eslint instead of jscs [#10]
+
+## hexo-renderer-stylus 1.0.0
+
+### Breaking change
+- chore: require at least node 8 [#33]
+
+### Fix
+- upgrade dependencies and make it be compatible with node v6 [#13]
```
Comment (YoshinoriN) [inline source/_posts/2019-09-06-hexo-generator-tag-1_hexo-renderer-ejs-1_hexo-renderer-stylus-1.md]: `upgrade dependencies and make it be compatible with node v6 [#13](https://github.com/hexojs/hexo-renderer-stylus/pull/13)` or add `[#13]: https://github.com/hexojs/hexo-renderer-stylus/pull/13` tail of markdown file.

**Comment by curbengh** [comment]
WIP, I would like to include hexo-renderer-stylus v1.1

**Pull request merged**